DirectXTex/BCDirectCompute.cpp

```diff
@@ -608,4 +608,4 @@ HRESULT GPUCompressBC::Compress(const Image& srcImage, const Image& destImage)
     }
 
     return hr;
-}
\ No newline at end of file
+}
```

Comment:
``-Wnewline-eof`` which would also be gripped about with ``-Wc++98-compat-pedantic``